resources/service-catalog/charts/catalog-crds/templates/metrics.yaml

```diff
@@ -0,0 +1,53 @@
+# Required because Service Catalog has Istio sidecar which Prometheus Operator doesn't have
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: {{ template "fullname" . }}
+spec:
+  peers:
+  - mtls:
+      mode: PERMISSIVE
+  targets:
+  - name: {{ template "fullname" . }}-controller-manager
+---
+# Dedicated Service for metrics endpoint
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  ports:
+  - name: http-metrics
+    port: 30444
+  selector:
+    app: {{ template "fullname" . }}-controller-manager
+---
+# Inform Prometheus to scrap the metrics endpoint
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    prometheus: monitoring
+    app: {{ template "fullname" . }}-controller-manager
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  name: {{ template "fullname" . }}-controller-manager
+spec:
+  endpoints:
+  - interval: 30s
+    port: secure
+    scheme: https
+    tlsConfig:
+      insecureSkipVerify: true
+  namespaceSelector:
+    matchNames:
+    - "{{ .Release.Namespace }}"
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}-controller-manager
```

Comment:
Service definition above should match this selectors? Or do you have some where else another service definition match this monitoring?

Reply:
I have another service definition which matches this selector in this file:
 ```kyma/resources/service-catalog/charts/catalog-crds/templates/controller-manager-service.yaml```